plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/ChatController.kt

```diff
@@ -85,6 +88,86 @@ class ChatController private constructor(
     private val messagePublisher: MessagePublisher = context.messagesFromAppToUi
     private val telemetryHelper = TelemetryHelper(context, chatSessionStorage)
 
+    init {
+        val scope = projectCoroutineScope(context.project)
+        context.project.messageBus.connect().subscribe(
+            CodeScanActionsListener.SEND_CODESCAN_ISSUE_TO_Q,
```

Comment:
this is not the right pattern to send commands to Q. Please take a look at how messages flow through `AmazonQApp` from a given `EditorContextCommand`

Reply:
This for sending messages from toolkit module to amazonq module. since the code scan issues are in toolkit module.